Mathlib/Algebra/Ring/Semireal/Defs.lean

```diff
@@ -0,0 +1,116 @@
+/-
+Copyright (c) 2024 Matematiflo. All rights reserved.
```

Comment:
```suggestion
Copyright (c) 2024 Florent Schaffhauser. All rights reserved.
```
Note sure if the linter is so smart to realize you didn't use your real name.

Reply:
Ok, I'll change this and remove the unnecessary import that was detected during CI checks.